pkg/collect/buffer_handler.go

```diff
@@ -63,6 +63,10 @@ func (h *CollectorHandler) add(m map[string]any) {
 func (h *CollectorHandler) Handle(ctx context.Context, r slog.Record) error {
 	m := make(map[string]any)
 
+	if TestID(ctx) != "" {
+		m["test_id"] = TestID(ctx)
+	}
+
 	m[slog.MessageKey] = r.Message
 
 	if h.addLevel {
```

Comment:
The collect package is only for testing, it is a "collector" handler.

Now it recognizes a context field "TestID" which when passed into the context it will attach it to the message as `test_id`. This is only for testing purposes.